basics/basics.ipynb

```diff
@@ -0,0 +1,323 @@
+    "</td>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3b439c7",
+   "metadata": {},
+   "source": [
+    "<td>\n",
+    "<a href=\"https://colab.research.google.com/github/Clarifai/examples/blob/main/datasets/upload/dataset_upload.ipynb\" target=\"_blank\"><img\n",
+    "src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Colab\"></a>\n",
+    "</td>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "89867234",
+   "metadata": {},
+   "source": [
+    "# Basics\n",
+    "\n",
+    "- Creating Clarifai Resources(App, Datasets)\n",
+    "- Interacting with Apps, Datasets\n",
+    "- Deleting App, Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cde4c58",
+   "metadata": {},
+   "source": [
+    "## Requirements"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4d9f747",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -U clarifai"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1a5ae2d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Replace your PAT\n",
+    "import os\n",
+    "os.environ['CLARIFAI_PAT'] = \"PAT\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b636e141",
+   "metadata": {},
+   "source": [
+    "*Note: Guide to get your [PAT](https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens)*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e87a9c1b",
+   "metadata": {},
+   "source": [
+    "## Getting User Client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6ed0b112",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Note: CLARIFAI_PAT must be set as env variable.\n",
+    "from clarifai.client.user import User\n",
+    "client = User(user_id=\"user_id\") #replace your \"user_id\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "baeb362b",
+   "metadata": {},
+   "source": [
+    "## Interacting with App"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91028bc4",
+   "metadata": {},
+   "source": [
+    "Applications are the basic building blocks for creating projects on the Clarifai platform. Your data, annotations, models, workflows, predictions, and searches are contained within applications. You can create as many applications as you want and edit or delete them as you see fit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e69b57b9",
+   "metadata": {},
+   "source": [
+    "### Creating App"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0d59c250",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create app and dataset\n",
+    "app = client.create_app(app_id=\"demo_app\", base_workflow=\"Universal\") #(Example Workflows: 'Universal', 'Empty', 'General')"
```

Comment:
This applies to all notebooks, can we have the comment above the code. Github doesn't provide horizontal scrolling in a cell and all of these comments just cuts off.